rfc822/mime.go

```diff
@@ -5,6 +5,9 @@ import (
 	"strings"
 )
 
+// ParseMediaType parses a MIME media type.
+var ParseMediaType = mime.ParseMediaType
```

Comment:
Why does this need to be a global variable?

Reply:
I'm not a big fan of this global variable to replace functionality. I Would rather have this as an input to the actual parsing function.

Reply:
That would require a much more significant change spanning multiple modules (here, bridge, liteapi). Doing it this way is quite common; we already do it in go-rfc5322, and it's done by other go libs as well (e.g. [here](https://github.com/emersion/go-message/blob/master/charset.go#L35))